rendering_styles/default.render.xml

```diff
@@ -3411,91 +3411,91 @@
 						<apply_if layer="-1" order="48"/>
 						<apply_if additional="layer=-2" order="24"/>
 					</case>
-					<case tag="highway" value="primary" order="82">
+					<case tag="highway" value="primary" order="82" renderingPriority="10">
 						<apply_if layer="1" order="105"/>
 						<apply_if additional="layer=2" order="135"/>
 						<apply_if additional="layer=3" order="163"/>
 						<apply_if layer="-1" order="60"/>
 						<apply_if additional="layer=-2" order="39"/>
 					</case>
-					<case tag="highway" value="primary_link" order="69">
+					<case tag="highway" value="primary_link" order="69" renderingPriority="8">
 						<apply_if layer="1" order="92"/>
 						<apply_if additional="layer=2" order="122"/>
 						<apply_if additional="layer=3" order="150"/>
 						<apply_if layer="-1" order="47"/>
 						<apply_if additional="layer=-2" order="23"/>
 					</case>
-					<case tag="highway" value="secondary" order="81">
+					<case tag="highway" value="secondary" order="81" renderingPriority="5">
 						<apply_if layer="1" order="104"/>
 						<apply_if additional="layer=2" order="134"/>
 						<apply_if additional="layer=3" order="162"/>
 						<apply_if layer="-1" order="59"/>
 						<apply_if additional="layer=-2" order="38"/>
 					</case>
-					<case tag="highway" value="secondary_link" order="68">
+					<case tag="highway" value="secondary_link" order="68" renderingPriority="5">
 						<apply_if layer="1" order="91"/>
 						<apply_if additional="layer=2" order="121"/>
 						<apply_if additional="layer=3" order="149"/>
 						<apply_if layer="-1" order="46"/>
 						<apply_if additional="layer=-2" order="22"/>
 					</case>
-					<case tag="highway" value="tertiary" order="80">
+					<case tag="highway" value="tertiary" order="80" renderingPriority="5">
 						<apply_if layer="1" order="103"/>
 						<apply_if additional="layer=2" order="133"/>
 						<apply_if additional="layer=3" order="161"/>
 						<apply_if layer="-1" order="58"/>
 						<apply_if additional="layer=-2" order="37"/>
 					</case>
-					<case tag="highway" value="tertiary_link" order="67">
+					<case tag="highway" value="tertiary_link" order="67" renderingPriority="5">
 						<apply_if layer="1" order="90"/>
 						<apply_if additional="layer=2" order="120"/>
 						<apply_if additional="layer=3" order="148"/>
 						<apply_if layer="-1" order="45"/>
 						<apply_if additional="layer=-2" order="21"/>
 					</case>
-					<case tag="highway" value="road" order="79">
+					<case tag="highway" value="road" order="79" renderingPriority="5">
 						<apply_if layer="1" order="102"/>
 						<apply_if additional="layer=2" order="132"/>
 						<apply_if additional="layer=3" order="160"/>
 						<apply_if layer="-1" order="57"/>
 						<apply_if additional="layer=-2" order="36"/>
 					</case>
-					<case tag="highway" value="unclassified" order="79">
+					<case tag="highway" value="unclassified" order="79" renderingPriority="5">
```

Comment:
default value not needed to be specified and better to make default value - 50 to have more flexibity cause this number is int.